nix/supervisord-cluster/default.nix

```diff
@@ -61,7 +70,7 @@ let
         addr = "127.0.0.1";
         port = p;
         valency = 1;
-      }) (lib.filter (p: p != selfPort) (lib.genList (i: basePort + i + 1) (composition.numBft + composition.numPools)));
+      }) (lib.filter (p: p != selfPort) (lib.genList (i: basePort + i + 1) (composition.n_bft_hosts + composition.n_pools)));
```

Comment:
Are we adopting new coding styles for variable names in nix?

Reply:
@newhoggy, this is mirroring the profile structure.